.github/workflows/main.yml

```diff
@@ -31,6 +32,11 @@ jobs:
 
       - name: Install dependencies
         run: composer install
+        if: matrix.php-version != '8.0'
+
+      - name: Update dependencies
+        run: rm composer.lock && composer remove --dev --no-update friendsofphp/php-cs-fixer && composer update
```

Comment:
Just FYI, it's often considered more readable to use multiple lines; otherwise it might also be easy to miss that there are 3 commands in total being executed:
```yaml
run: |
  rm composer.lock
  composer remove --dev --no-update friendsofphp/php-cs-fixer
  composer update
```
This works quite well without `&&` because github runs actions with `bash -e` which immediately exits on a failing command, i.e. if `composer remove…` would fail for some reason, the behaviour would be the same with the long line and `&&`